src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -554,7 +554,11 @@ export const getInitialSwedishHouseValues = (
     numberOfBathrooms: details.numberOfBathrooms,
     yearOfConstruction: details.yearOfConstruction,
     isSubleted: details.isSubleted,
-    extraBuildings: details.extraBuildings,
+    extraBuildings: details.extraBuildings.map((building) => ({
+      type: building.type,
+      area: building.area,
+      hasWaterConnected: building.hasWaterConnected,
+    })),
```

Comment:
When we're at it - could we maybe do a destructuring here? 
```suggestion
    extraBuildings: details.extraBuildings.map(({type, area, hasWaterConnected}) => ({
      type,
      area,
      hasWaterConnected,
    })),
```